modules/latex.php

```diff
@@ -91,13 +339,23 @@ function latex_shortcode( $atts, $content = '' ) {
 }
 
 /**
- * LaTeX needs to be untexturized
+ * LaTeX needs to be untexturized.
+ *
+ * @since 1.1
+ *
+ * @param  array $shortcodes
+ * @return array
  */
 function latex_no_texturize( $shortcodes ) {
 	$shortcodes[] = 'latex';
 	return $shortcodes;
 }
 
+add_filter( 'wp_head', 'latex_mathjax_config', 1 );
```

Comment:
Why does `latex_mathjax_config()` need to be attached to `wp_head` when its called by `latex_mathjax_output()` anyway?